frontend/src/Componentes/Tarjeta/Tarjeta.jsx

```diff
@@ -1,4 +1,4 @@
-const TarjetaCowboy = () => {
+const TarjetaTonta = ({ nombre, titulos, titulosCantidad, total }) => {
```

Comment:
Es mejor que el propio componente muestre el total de títulos y los títulos separados por coma.
Mejor solamente pasar nombre, títulos y total.


```suggestion
const TarjetaTonta = ({ nombre, titulos, total }) => {
```